### DISCUSSION TITLE
Vehicle Gifting

### DISCUSSION THREAD

**Discussion by Deadlineem** [Deadlineem/Extras-Addon-for-YimMenu#7] [Q&A]
### -- Requirements --
MAKE SURE ALL PERSONAL VEHICLES ARE REPAIRED/RETURNED TO STORAGE!!

Receiving player needs to have a full garage of burner cars (can be bought FREE from Southern San Andreas Website by clicking 2 Door then clicking Price).

Receiving player needs to enter the full garage and drive one of the burner cars out/back in to ensure it doesn't overwrite their last used personal vehicle.

-----------------------------------------------------
### How To Gift Vehicles
-----------------------------------------------------
Spawn a vehicle using Extras Addon's Vehicle Spawner (NOT the spawner built into YimMenu!)

(Optional).  Modify their vehicle for them, with custom wheels, paints, etc.

### If the player has met the requirements, have them get into the vehicle and then you spam 'Gift Vehicle' until it reads success

Have the receiving player drive in and replace a burner car in their garage

-----------------------------------------------------

The vehicles come fully insured regardless of which way you are doing it.  If the player gets kicked from the vehicle after you mod it using Option 1, try spawning the vehicle with the Extras Addon spawner and not modding the vehicle at all.  The gifting process should work then.

**MAKE SURE YOU USE THE DELETE GUN TO DELETE THE PHANTOM VEHICLES THAT ARE THERE AFTER THEY DRIVE INTO THE GARAGE!!!**

Self > Weapons > Custom Gun (checked) > Dropdown > Delete Gun - Point/Shoot

**Comment by vyralove** [comment]
As of 7/6/2024:
Don't want to report an issue, so asking/confirming for clarification:
We followed the instructions on the tooltip of the Extras Addon's Vehicle Spawner (I am the receiver). Tried both a 10 car garage and a 10 car garage attached to an apartment complex. Both times this did not work.
I would either get kicked out of the vehicle or the garage would refuse entry and tell me to purchase the vehicle.
We are at a loss on how to properly do this despite following proper procedure. (Also did not want to make a new topic for this when this thread exists.)

Steps:
Fill garage with free burner cars.
Take one out/drive it back in to reset personal vehicle.
Vehicle spawns in. I get in and drive it close to my garage entrance.
Get gifted the vehicle.
????
I get kicked from vehicle/garage won't let me put it in. (Vehicles were either un-upgraded or fully upgraded. Both times, did not work.)

**Reply by Deadlineem** [reply]
The vehicle gifting in ALL MENUS has been patched with GTA Update 1.69, I am working on a solution to fix this but as of current there is no estimated time that this will be working.

**Reply by vyralove** [reply]
Ah, okay. That's fine. Take your time and keep up the good work. :) I had a feeling it got patched.

**Reply by Deadlineem** [reply]
Yeah, it sucks too because it took me about a month total to get it to work and R* goes and patches the shit.

**Comment by real1983** [comment]
I have a question is this safe like updated for current version of gta v 1.69? and gifting cars does not work currently thanks in advanced

**Reply by Deadlineem** [reply]
> I have a question is this safe like updated for current version of gta v 1.69? and gifting cars does not work currently thanks in advanced

Its safe it just causes you to crash when using it more often than not and gifting currently does not work at all.
